### ISSUE TITLE
grunt serve command not working

### ISSUE THREAD

**Issue by thomasbritton** [leemunroe/grunt-email-workflow#112]
When I run `grunt serve` it gives me the following error:

```
Loading "express.js" tasks...ERROR
>> TypeError: os.tmpDir is not a function
Warning: Task "express" not found. Use --force to continue.

Aborted due to warnings.
```

Any ideas? I am on Windows 10.

Thanks

**Comment by taeo** [comment]
Hey @thomasbritton 

Try deleting your `node_modules` folder, then re-installing `npm install`.

If that's not working for some reason, can you try giving this branch a run [fix/2020-07-package-audit](https://github.com/leemunroe/grunt-email-workflow/tree/fix/2020-07-package-audit)

**Comment by thomasbritton** [comment]
Hey @taeo 

I've just tried using the branch as suggested and it reports back with the same error message as first reported.

**Comment by leemunroe** [comment]
@thomasbritton I'm seeing same issue after upgrading to Node 14.8.0

@taeo Tried [your branch](https://github.com/leemunroe/grunt-email-workflow/tree/fix/2020-07-package-audit) and got the same issue. 

The dependency versions looked up to date in that branch. It wasn't obvious to me where the issue is. I didn't have time to dig in further.

**Comment by thomasbritton** [comment]
@leemunroe I'm on Node 14.5.0

**Comment by taeo** [comment]
@leemunroe and @thomasbritton 

Ok. Definitely a node version issue. In general, good practice to run [LTS versions](https://nodejs.org/en/about/releases/) (at this time that's v10 and v12).

Code updates not withstanding, here's the fix process to get running ([see related commit](https://github.com/leemunroe/grunt-email-workflow/commit/1d7cdef9e0e103aaeec3c62771054b3eb54ebca3)).

1. Install Node Version Manager (NVM) - [Mac / Linux](https://github.com/nvm-sh/nvm) vs [Windows](https://github.com/coreybutler/nvm-windows)
2. `nvm install lts/Erbium` - Install 12x node version
3. `nvm use` - will set correct node version
4. `rm -rf node_modules` - clean house
5. `npm install` - ensure clean npm packages for corrected node version
6. Run your scripts as usual

**Comment by leemunroe** [comment]
Yes this worked for me. Thanks @taeo!

tl;dr don't use 14.x yet ([release schedule](https://nodejs.org/en/about/releases/))

**Comment by thomasbritton** [comment]
@taeo works great now! 

Thanks
